actions/ci-beholder-validator/README.md

```diff
@@ -0,0 +1,115 @@
+# Schema Registry Validator Action
+
+A GitHub Action that validates schema compatibility against your schema registry
+during pull requests. This action helps prevent breaking changes by
+automatically checking schema evolution compatibility before changes are merged.
+
+## Overview
+
+The Schema Registry Validator action automatically:
+
+1. Spins up a local Redpanda schema registry for testing
+2. Validates your schemas against it
+3. Ensures schema changes maintain backward compatibility
+4. Reports validation results in your PR
+
+## Prerequisites
+
+- GitHub repository with Actions enabled
+- Schema files (.proto, .avsc, etc.) in your repository
+- A `beholder.yaml` configuration file
+
+## Usage
+
+Add this action to your workflow:
+
+```yaml
+name: Validate Schemas
+on:
+  pull_request:
+    paths:
+      - "**.proto"
+      - "**.avsc"
+      - "**/beholder.yaml"
+
+jobs:
+  validate:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: smartcontractkit/ci-beholder-validator@v1
```

Comment:
If this is supposed to reference this action it will look more like:

`uses: smartcontractkit/.github/actions/ci-beholder-validator@<sha> # ci-beholder-validator@1.0.0`

But if you're hoping to have a @v1 tag then you will need a separate repository that syncs this action

Reply:
Let me change this

Reply:
Does the new change look okay? In some cases I am only seeing sha and in some cases I am seeing versions.

Reply:
The tag that will be created in this repo will be `ci-beholder-validator@1.0.0`. Which means the reference will need to be:

```
smartcontractkit/.github/actions/ci-beholder-validator@<sha> # ci-beholder-validator@1.0.0
```

It will need to use a sha because you cannot reference a tag with an `@`

Reply:
Done